cli/cmd/release/integrate.go

```diff
@@ -0,0 +1,150 @@
+package release
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+	"github.com/wordpress-mobile/gbm-cli/internal/repo"
+	"github.com/wordpress-mobile/gbm-cli/internal/utils"
+	"github.com/wordpress-mobile/gbm-cli/pkg/release"
+)
+
+var (
+	Ios        bool
+	Android    bool
+	Update     bool
+	BaseBranch string
+	Verbose    bool
+	tempDir    string
+)
+
+func cleanup() {
+	os.RemoveAll(tempDir)
+}
+
+func init() {
+	// Make sure we clean up temp files on early exits
+	// Use a buffered channel so we don't miss the signal.
+	// see https://go.dev/tour/concurrency/5 and https://gobyexample.com/signals
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
+		cleanup()
+		os.Exit(1)
+	}()
+}
+
+func setTempDir() {
+	var err error
+	if tempDir, err = ioutil.TempDir("", "gbm-"); err != nil {
+		fmt.Println("Error creating temp dir")
+		os.Exit(1)
+	}
+}
+
+// checklistCmd represents the checklist command
+var IntegrateCmd = &cobra.Command{
+	Use:   "integrate <version>",
+	Short: "generate the release integration Pr",
+	Long: `
+`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		version := args[0]
+		gbmPr, err := utils.GetGbmReleasePr(version)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		type result struct {
+			repo string
+			pr   repo.PullRequest
+			err  error
+		}
+		rChan := make(chan result)
+
+		s := spinner.New(spinner.CharSets[23], 200*time.Millisecond)
+
+		setTempDir()
+		defer cleanup()
+
+		// if neither ios or android are specified, default to both
+		if !Ios && !Android {
+			Ios = true
+			Android = true
+		} else {
+			// if we are only doing one, set verbose to true
+			Verbose = true
+		}
+		numPr := 0 // number of PRs to create
+
+		// Use goroutines to create the PRs concurrently
+		if Ios {
+			numPr++
+			utils.LogInfo("Creating iOS PR at %s/Wordpress-iOS", repo.WpMobileOrg)
+			go func() {
```

Comment:
Running each PR set up in a goroutine allow for parallelizing the work. 
I'm not sure how but the output from each "job" is synchronous. 
It could be how go captures the output or it could somehow be running the work in series. 
I still need to investigate